django_project/geohosting/views/products.py

```diff
@@ -22,9 +22,6 @@
     fetch_erpnext_data,
     fetch_erpnext_detail_data
 )
-from geohosting_controller.default_data import (
```

Comment:
Hi @tinashechiraya 
Is there any reason there are deleted lines that is not related to ticket?